src/e-cherry-pick.js

```diff
@@ -131,16 +135,22 @@ program
       patchUrlStr = targetBranch;
       targetBranch = tmp;
     }
+
     const octokit = new Octokit({
```

Comment:
I think we can drop octokit completely from this flow, we only use it to check permissions now which is not required.

We should load the clone url from `config.remotes.electron.origin` and then if the config is using a fork we'll just push to that (`config.remotes.electron.fork`) instead.

Then instead of using octokit to make a PR (because this device code workflow sucks) we just open them to the pull request create page in the same way `e pr` does.

Then the whole thing works for forks (yay) and with no github auth dance (yay)